custom_components/variable/sensor.py

```
@@ -458,38 +476,45 @@ async def async_increment_variable(self, **kwargs) -> None:
+                new_value = int(new_value)
 
-            _LOGGER.debug(f"({self._attr_name}) [async_increment_variable] New Value: {new_value}")
+            _LOGGER.debug(
+                "(%s) [async_increment_variable] New Value: %s", self._attr_name, new_value
+            )
             self._attr_native_value = new_value
             self.async_write_ha_state()
 
         except ValueError as err:
-            _LOGGER.error(f"({self._attr_name}) Increment error: {err}")
+            _LOGGER.error("(%s) Increment error: %s", self._attr_name, err)
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)